### DISCUSSION TITLE
Adjusting dot width

### DISCUSSION THREAD

**Discussion by merkkat** [kudrykv/latex-yearly-planner#137] [Q&A] [answered]
Hello! 

I'm wondering if it's possible to adjust the dot width to 4mm? I see the line height that I can change in the base.yaml but that doesn't seem to change the dot width.

**Comment by kudrykv** [comment, marked as answer]
Withing the existing code, you may try tweaking [`myDotGrid` command](https://github.com/kudrykv/latex-yearly-planner/blob/2dc9ab8cd32e8ad7eaff11ab99c888891022e77f/tpls/macro.tpl#L79)

If you're using MOS layout, you may try [work-in-progress, code re-do](https://github.com/kudrykv/latex-yearly-planner/tree/rubify2) (same planner, easier-to-maintain code). Although being WIP, it starts to solidify already.

While pattern isn't configurable (yet, at least), you can change it [in the index template file](https://github.com/kudrykv/latex-yearly-planner/blob/0f8685e803e0c83e26518a21d4a74ffc0be3db61/lib/latex_yearly_planner/erb/index.typ.erb#L18).
